Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start with a Linux micro-container to keep the image tiny
+FROM alpine:3.3
+
+# Install python and dependencies
+RUN apk add --update \
+	python-dev \
+    py-pip \
+    postgresql-dev \ 
+    gcc \ 
+    musl-dev \
+ && rm -rf /var/cache/apk/*
+
+# Expose any ports the app is expecting in the environment
+ENV PORT 5000
+EXPOSE $PORT
+
+# Get connection string from build-args
+ARG URI
+ENV LOCAL_DB=$URI
+
```

Comment:
@jdgumz this got lost earlier. meant to raise the question again. don't htink your solution uses this bit at all, does it? if not, doesn't seem to have a negative effect, but should we remove if not needed?